Source/ACE.Entity/AceObject.cs

```diff
@@ -296,7 +283,7 @@ public string Name
         /// </summary>
         public uint ItemType
         {
-            get { return GetIntProperty(PropertyInt.ItemType).Value; }
+            get { return IntProperties.Find(x => x.PropertyId == (uint)PropertyInt.ItemType).PropertyValue; }
```

Comment:
There was a handful of them, I only commented on one.  We'll get the others later.